nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/CandidateBO.java

```diff
@@ -348,6 +343,34 @@ private static String mapToUsernameString(Username assignee) {
         return assignee != null ? assignee.value() : null;
     }
 
+    private void validateCandidateState() {
+        if (periodStatus.status().equals(Status.CLOSED_PERIOD)) {
+            throw new IllegalStateException(PERIOD_CLOSED_MESSAGE);
+        }
```

Comment:
I know that i added this if statement, but it should be Statsus.CLOSED_PERIOD.equals(...